app/src/main/res/layout/auth_ui_layout.xml

```diff
@@ -1,235 +1,242 @@
 <?xml version="1.0" encoding="utf-8"?>
-<ScrollView
+<android.support.design.widget.CoordinatorLayout
```

Comment:
Thanks for all that!  I realized I was confusing `CoordinatorLayout` and `ConstraintLayout` in my head!  First day back fuzz I guess.